src/modules/75-cf/pages/feature-flags-detail/targeting-rules-tab/TargetingRulesTab.tsx

```diff
@@ -179,7 +208,24 @@ const TargetingRulesTab = ({
       initialValues={initialValues}
       validate={values => validate(values)}
       onSubmit={values => {
-        saveChanges(values)
+        if (initialValues.state === 'on') {
+          if (initialValues.onVariation != values.onVariation) {
```

Comment:
Super minor, but try to always default to type-safe comparisons unless you have a specific use-case for type-agnostic comparison.
```suggestion
          if (initialValues.onVariation !== values.onVariation) {
```

Reply:
Great spot, thanks have fixed